cura/Api/Backups.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+from cura.Backups.BackupsManager import BackupsManager
+
+
+class Backups:
+    """
+    The backups API provides a version-proof bridge between Cura's BackupManager and plugins that hook into it.
+
+    Usage:
+        from cura.Api import CuraApi
```

Comment:
I would capitalize API in all cases where possible. e.g. from `cura.API import CuraAPI`

Reply:
Fixed.